python/lsst/ts/tunablelaser/canbus_modules.py

```diff
@@ -42,20 +42,25 @@
     "LDCO48BP",
     "MLDCO48",
     "DelayLin",
+    "MidiOPG",
 ]
 import logging
 
-from .ascii import AsciiRegister
+from .register import AsciiRegister
 from .enums import Mode, NoSCU, Output, Power, SCUConfiguration
 
 
 class CPU8000:
```

Comment:
It looks like all these laser classes have very similar constructors (both in terms of arguments and functionality) and at least share the `update_register` method. Please extract a common Abstract Base Class (ABC) that sets up the registers and make sure that all these laser classes subclass that ABC.